### DISCUSSION TITLE
Tasmota  sonoff POW R3

### DISCUSSION THREAD

**Discussion by novembre62** [arendst/Tasmota#18191] [Q&A]
Hi

 I have a sonoff POW R3 with 'Tasmota 12.4.0

 everything works regularly and is set with the relay ON at start-up.

 It happens that by manually turning the relay OFF and leaving it like this,  periodically (after  a few minutes...not yet quantized), the relay switches by itself to ON and then back to OFF again in less than a second...the thing then repeats itself.

 While when it is set to ON this does not happen.

 I can't understand why

**Comment by sfromis** [comment]
Could be electrical noise. I'd suggest activating `Weblog 3` and then check the console after it happened, to verify the source.

**Reply by novembre62** [reply]
I activated Weblog 3 and when the problem occurs the console:

00:00:00.351 WIF: Attempting connection...
00:00:00.688 WIF: Connecting to AP1 TIM-2919xxxxx Channel 6 BSSId xx:xx:xx:xx:xx:xx in mode 11n as tasmota-503DEC-7660...
00:00:01.125 WIF: Attempting connection...
00:00:02.131 WIF: Attempting connection...
00:00:03.162 WIF: Attempting connection...
00:00:04.122 WIF: Attempting connection...
00:00:05.754 WIF: Attempting connection...
00:00:06.477 QPC: Reset
00:00:06.753 WIF: Attempting connection...
00:00:07.781 WIF: Connected
00:00:08.003 HTP: Web server active on tasmota-503DEC-7660 with IP address 192.168.1.42
00:00:08.007 RSL: INFO1 = {"Info1":{"Module":"Sonoff POWR3","Version":"12.4.0(tasmota)","FallbackTopic":"cmnd/DVES_503DEC_fb/","GroupTopic":"cmnd/tasmotas/"}}
00:00:08.010 RSL: INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"tasmota-503DEC-7660","IPAddress":"192.168.1.42"}}
00:00:08.012 RSL: INFO3 = {"Info3":{"RestartReason":{"Exception":4,"Reason":"Software Watchdog","EPC":["40003b53","00000000","00000000"],"EXCVADDR":"00000000","DEPC":"00000000"},"BootCount":454}}
00:00:08.014 RSL: RESULT = {"POWER":"OFF"}
00:00:08.016 RSL: POWER = OFF
00:00:08.401 APP: Boot Count 454
00:00:08.432 WIF: DNS resolved 'pool.ntp.org' (37.247.53.178) in 28 ms
00:00:08.476 RTC: UTC 2023-03-14T16:28:03, DST 2023-03-26T02:00:00, STD 2023-10-29T03:00:00
17:28:03.000 RTC: Synced by NTP
17:28:03.084 CFG: Saved to flash at F8, Count 491, Bytes 4096
